### PR TITLE
throw datadog statistics on transcription events

### PR DESCRIPTION
**Pull request by nikvaessen**

### CODE REVIEW

File: src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java [hunk 1]
```diff
@@ -154,6 +155,16 @@
     private static final Logger logger
         = Logger.getLogger(AbstractTranscriptPublisher.class);
 
+    /**
+     * Ascept for successful upload of transcript
+     */
+    private static final String DD_ASCEPT_SUCCESS = "upload_success";
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java]: Aspect?

File: src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java [hunk 2]
```diff
@@ -154,6 +155,16 @@
     private static final Logger logger
         = Logger.getLogger(AbstractTranscriptPublisher.class);
 
+    /**
+     * Ascept for successful upload of transcript
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java]: "Upload" refers to the execution of the external script after the call finishes, correct?

File: src/main/java/org/jitsi/jigasi/transcription/Transcriber.java [hunk 3]
```diff
@@ -42,6 +44,16 @@
...
 
+    /**
+     * Datadog ascept for starting transcribing
+     */
+    private final static String DD_ASPECT_START = "start_transcriber";
+
+    /**
+     * Datadog ascept for ending transcribing
+     */
+    private final static String DD_ASCEPT_STOP = "stop_transcriber";
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jigasi/transcription/Transcriber.java]: Aspect?

File: src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java (hunk 2 above)
Comment (nikvaessen) [reply src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java]: yes

**Pull request merged**